std/conv.d

```diff
@@ -5588,6 +5588,19 @@ template castFrom(From)
     }
 }
 
+// instantiate the documenting unittest in the castFrom template
+unittest
+{
+    alias castFromI = castFrom!long;
```

Comment:
Yes, can't we just move the unittest outside? I think it will just document the "castFrom" template instead of the "to" member, but that should be fine.